# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/ipc/ContentParent.cpp
# Commit: 241ea498604b
# Full Hash: 241ea498604b8b8057b4e09aa5e9f4a89fe70e98
# Author: Kris Maglione <maglione.k@gmail.com>
# Date: 2020-04-21 02:08:37
# Description:
#   Bug 1629477: Remove failing assertion. r=nika
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D71369
# ==============================================================================

diff -r f352804ccc8d -r 241ea498604b dom/ipc/ContentParent.cpp
--- a/dom/ipc/ContentParent.cpp	Tue Apr 21 01:11:37 2020 +0300
+++ b/dom/ipc/ContentParent.cpp	Mon Apr 20 15:29:40 2020 +0000
@@ -872,7 +872,6 @@
       (p = PreallocatedProcessManager::Take()) && !p->mShutdownPending) {
     // For pre-allocated process we have not set the opener yet.
     p->mOpener = aOpener;
-    MOZ_DIAGNOSTIC_ASSERT(p->mScriptableHelper);
     aContentParents.AppendElement(p);
     p->mActivateTS = TimeStamp::Now();
     return p.forget();
